packages/edit-site/src/components/style-book/index.js

```diff
@@ -332,19 +337,22 @@ const StyleBookBody = ( {
 				}
 				isSelected={ isSelected }
 				onSelect={ onSelect }
+				key={ category }
 			/>
 		</Iframe>
 	);
 };
 
 const Examples = memo(
 	( { className, examples, category, label, isSelected, onSelect } ) => {
-		const composite = useCompositeState( { orientation: 'vertical' } );
+		const compositeStore = useCompositeStore( { orientation: 'vertical' } );
+
 		return (
 			<Composite
-				{ ...composite }
+				store={ compositeStore }
 				className={ className }
 				aria-label={ label }
+				role="grid"
```

Comment:
What's the rationale for changing the role to `grid`? Does it make sense even if every row always contains only one cell?

Reply:
> The grid pattern can be used to group a set of interactive elements, such as links, buttons, or checkboxes. Since only one element in the entire grid is included in the tab sequence, grouping with a grid can dramatically reduce the number of tab stops on a page. [^1]

In theory it should make it clearer that the buttons (as rendered by `Example` below) should be navigated with arrow keys rather than tab. Most screen readers will announce the number of rows too, which means users get told how many buttons are in the collection

[^1]: [W3: Layout Grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/#layoutgridsforgroupingwidgets)

Reply:
Not a blocker for this PR, but — have you considered using `CompositeRow` (and potentially `CompositeGroup`) ?

Reply:
I did, but I didn't want to move this too far away from the original implementation.